plugins/woocommerce/client/legacy/js/jquery-blockui/jquery.blockUI.js

```diff
@@ -191,7 +191,7 @@
 			// enable if you want key and mouse events to be disabled for content that is blocked
 			bindEvents: true,
 
-			// be default blockUI will supress tab navigation from leaving blocking content
+			// be default blockUI will suppress tab navigation from leaving blocking content
```

Comment:
Seems like there's another typo here, `be` for `by`?

```suggestion
			// by default blockUI will suppress tab navigation from leaving blocking content
```